examples/constant.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2015 The Noise-rs Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! An example of generating constant valued noise
+
+extern crate noise;
+
+use noise::modules::Constant;
+
+mod debug;
+
+fn main() {
+    debug::render_png2("constant1.png", Constant::new(-1.0), 1024, 1024, 1);
+    debug::render_png2("constant2.png", Constant::new(0.0), 1024, 1024, 1);
+    debug::render_png2("constant3.png", Constant::new(1.0), 1024, 1024, 1);
+
+    println!("Generated constant1.png, constant2.png, and constant3.png");
```

Comment:
Maybe move this logging to the end of the `render_png2` and `render_png` functions. That way you get one for each file as it processes.